lib/rps_game.rb

```diff
@@ -9,8 +9,8 @@ def initialize(player1, player2 = "Computer")
   end
 
   def play(player_1_choice, player_2_choice = computer_choice)
-    raise 'Invalid Input by Player' unless SELECTION.include?(player_1_choice)
-    raise 'Invalid Input by Player 2' unless SELECTION.include?(player_2_choice)
+    raise "Invalid Input by #{@player1}" unless SELECTION.include?(player_1_choice)
```

Comment:
I'm a big fan of how thorough you are with your error messages - never trust users to use your program properly!